From 98187a5238939edc889fdaad93b00628b55d1e08 Mon Sep 17 00:00:00 2001
From: Maxime Hadjinlian <maxime.hadjinlian@gmail.com>
Date: Sat, 3 May 2014 19:43:20 +0200
Subject: [PATCH] test-serial: Remove test-serial

This folders only contains python tests scripts which would force us to
enable a dependency to host-python which seems kind of silly since
theses are only tests scripts.

Signed-off-by: Maxime Hadjinlian <maxime.hadjinlian@gmail.com>
---
 Makefile.am  | 2 +-
 configure.ac | 6 ------
 2 files changed, 1 insertion(+), 7 deletions(-)

diff --git a/Makefile.am b/Makefile.am
index 265f0d7..e7bb1a1 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -2,7 +2,7 @@ include $(top_srcdir)/Makefile.common
 
 ACLOCAL_AMFLAGS = -I m4
 
-SUBDIRS = lib addrdb src tests test-serial
+SUBDIRS = lib addrdb src tests
 
 include_HEADERS = include/ieee802154.h include/nl802154.h
 noinst_HEADERS = include/libcommon.h include/addrdb.h include/logging.h
diff --git a/configure.ac b/configure.ac
index 17658af..506b289 100644
--- a/configure.ac
+++ b/configure.ac
@@ -77,11 +77,6 @@ AC_PROG_LIBTOOL
 AM_PROG_LEX
 AC_PROG_YACC
 AC_SUBST(LIBTOOL_DEPS)
-# Currently the scripts works with python2.x only
-# Check for this at first
-AC_PATH_PROG(PYTHON, python2 python2.7 python2.6 python2.5 python)
-AM_PATH_PYTHON(,, [:])
-py_CHECK_MAJOR_VERSION([PYTHON_VERSION], [2])
 AM_MISSING_PROG(HELP2MAN, help2man, $missing_dir)
 
 # Checks for libraries.
@@ -135,6 +130,5 @@ AC_CONFIG_FILES([
 	addrdb/Makefile
 	src/Makefile
 	tests/Makefile
-	test-serial/Makefile
 ])
 AC_OUTPUT
-- 
1.9.2

